newIDE/app/src/UI/LeftLoader.js

```diff
@@ -14,7 +14,15 @@ const LeftLoader = ({
   isLoading: ?boolean,
 }) => (
   <span>
-    {isLoading && <CircularProgress size={20} style={styles.progress} />}
+    {isLoading && (
+      <CircularProgress
+        // From size 20, this component applied to a Dialog button triggers glitches
```

Comment:
We might want to add some padding?

Reply:
As discussed together, the padding adds scroll to the entire dialog